content/en/account_management/guide/hourly-usage-migration.md

```diff
@@ -0,0 +1,415 @@
+          {
+            "usage_type": "agent_host_count",
+            "value": 1
+          },
+          {
+            "usage_type": "alibaba_host_count",
+            "value": 2
+          },
+          {
+            "usage_type": "apm_azure_app_service_host_count",
+            "value": 3
+          },
+          {
+            "usage_type": "apm_host_count",
+            "value": 4
+          },
+          {
+            "usage_type": "aws_host_count",
+            "value": 5
+          },
+          {
+            "usage_type": "azure_host_count",
+            "value": 6
+          },
+          {
+            "usage_type": "container_count",
+            "value": 7
+          },
+          {
+            "usage_type": "gcp_host_count",
+            "value": 8
+          },
+          {
+            "usage_type": "heroku_host_count",
+            "value": 9
+          },
+          {
+            "usage_type": "host_count",
+            "value": 10
+          },
+          {
+            "usage_type": "infra_azure_app_service",
+            "value": 11
+          },
+          {
+            "usage_type": "opentelemetry_host_count",
+            "value": 12
+          },
+          {
+            "usage_type": "vsphere_host_count",
+            "value": 13
+          }
+        ],
+        "product_family": "infra_hosts"
+      },
+      "type": "usage_timeseries",
+      "id": "ec3e0318b98d15c2ae8125e8bda0ff487cd08d80b120fb340c9322ee16f28629"
+    }
+  ]
+}
+```
+
+#### Notes
+
+* Objects in the data array represent hourly usage, for each product and each organization.
+    * V1 APIs did not support multiple products or multiple organizations per request.
+* Usage measurements are represented in the nested `measurements` array.
+* Usage measurement objects have the fields `usage_type` and `value`.
+* `hour`, `org_name`, and `public_id` are also fields in the `attributes` object.
+
+## Pagination
+
+The v2 hourly usage API is paginated. Responses are limited to 500 pages, with a page containing usage data for one
+product family, for one hour, for one organization. Pagination allows the API to support other features such as multiple
+products per request, multiple organizations per request, and unlimited time ranges.
+
+If a result has more pages, the record ID of the next page is returned in the field
+`meta.pagination.next_record_id`. Clients should then pass that id in the parameter `pagination[next_record_id]`. There
+are no more pages to retrieve when the `meta.pagination.next_record_id` field is not set.
+
+### Pseudo-code example
+```
+response := GetHourlyUsage(start_time, end_time, product_families)
+cursor := response.metadata.pagination.next_record_id
+WHILE cursor != null BEGIN
+sleep(5 seconds)  # Avoid running into rate limit
+response := GetHourlyUsage(start_time, end_time, product_families, next_record_id=cursor)
+cursor := response.metadata.pagination.next_record_id
+END
+```
+
+## Multi-organization responses
```

Comment:
@RolfBates @buraizu also recommend updating this section to add a note about multi-region support as well